rsconnect/bundle.py

```diff
@@ -118,10 +120,17 @@ def __init__(
                 },
             }
 
-        if image:
-            self.data["environment"] = {
-                "image": image,
-            }
+
+        if image or env_management_py is not None or env_management_r is not None:
+            self.data["environment"] = {}
+            if image:
+                self.data["environment"]["image"] = image
+            if env_management_py is not None or env_management_r is not None:
+                self.data["environment"]["environment_management"] = {}
+                if env_management_py is not None:
+                    self.data["environment"]["environment_management"]["python"] = env_management_py
+                if env_management_r is not None:
+                    self.data["environment"]["environment_management"]["r"] = env_management_r
```

Comment:
This block seems to be present 3 (as far as I've seen) times. Is it worth putting it in a small helper function e.g. `make_environment_data(image, env_management_py, env_management_r)`? I don't have the larger context of this file and how manifest generation is factored out so it may not make sense, but worth asking.